### PR TITLE
in-progress

### PR DESCRIPTION
**Pull request by kami-delaroz**
_Replace me with a description of the changes in this PR_

---

_Additional context for reviewer_

---
_Release Notes_: 
_Replace me with a short sentence that describes the effect of this change on Salto users_

---
_User Notifications_: 
_Replace me with a short sentence that describes changes that will appear in NaCls and are not caused by user actions (e.g. a new annotation, field values that are converted to references, etc). Hidden changes should not be listed._

### CODE REVIEW

File: packages/core/src/api.ts [hunk 1]
```diff
@@ -242,6 +242,7 @@ const updateStateWithFetchResults = async (
   workspace: Workspace,
```
Comment (kami-delaroz) [inline packages/core/src/api.ts:242]: nothing from this file

File: packages/core/src/local-workspace/state.ts [hunk 2]
```diff
@@ -319,9 +319,9 @@ export const localState = (
       await inMemState.overridePathIndex(unmergedElements)
```
Comment (kami-delaroz) [inline packages/core/src/local-workspace/state.ts:319]: nothing from here

File: packages/core/test/common/elements.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ export const getAllElements = (): AllElementsTypes => {
       city: { refType: BuiltinTypes.STRING },
```
Comment (kami-delaroz) [inline packages/core/test/common/elements.ts:26]: nothing from here

File: packages/workspace/src/workspace/path_index.ts [hunk 4]
```diff
@@ -201,13 +201,15 @@ export const overrideTopLevelPathIndex = async (
 type UpdateIndexParams = {
```
Comment (kami-delaroz) [inline packages/workspace/src/workspace/path_index.ts:201]: nothing from here

File: packages/workspace/src/workspace/state/state.ts [hunk 5]
```diff
@@ -60,7 +60,7 @@ export interface State extends ElementsSource {
   getServicesUpdateDates(): Promise<Record<string, Date>>
   existingAccounts(): Promise<string[]>
```
Comment (kami-delaroz) [inline packages/workspace/src/workspace/state/state.ts:61]: nothing from here

**Comment by ori-moisis** [comment]
@kami-delaroz what is this PR? do we need it? can we close it?

**Comment by kami-delaroz** [comment]
it's my previous work on partial deletion in core side.
will close

**Pull request closed without merging**